pkg/pxc/app/statefulset/node.go

```diff
@@ -136,6 +136,12 @@ func (c *Node) AppContainer(spec *api.PodSpec, secrets string, cr *api.PerconaXt
 					SecretKeyRef: app.SecretKeySelector(secrets, "monitor"),
 				},
 			},
+			{
+				Name: "POD_NAMESPACE",
+				ValueFrom: &corev1.EnvVarSource{
```

Comment:
adding new vars adds unnecessary restarts of database during the update of the operator.
can you please take namespace from `/run/secrets/kubernetes.io/serviceaccount/namespace` file inside `build/pxc-entrypoint.sh` without adding any new env variables